### PR TITLE
add pyocct

### PR DESCRIPTION
**Pull request by looooo**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyocct/build.sh [hunk 1]
```diff
@@ -2,8 +2,9 @@
 mkdir build
 cd build
 
-# export CC=gcc-4.9
-# export CXX=g++-4.9
+if [[ ${HOST} =~ .*linux.* ]]; then
+    CXXFLAGS="${CXXFLAGS} -fpermissive"
```
Comment (trelau) [inline recipes/pyocct/build.sh]: @looooo should this be a flag in the main pyOCCT CMake?

**Comment by looooo** [comment]
@trelau Vc 14 feature is not needed anymore.

**Comment by trelau** [comment]
@looooo ah...do you know if there is a way to disable and/or not use the appveyor build? it times out at 60 min...i had to put in a special request to extend that time for the LaughlinResearch/pyOCCT account...i assume we can just use the azure windows build?

**Comment by looooo** [comment]
@trelau  renaming the `.appveyor`-file seems to work (disabling appveyor). I also think it makes sense to build only against p37 for now and enable older versions once we have a working linux package. 

And sorry for stopping the windows build. I didn't expect the build still running...

**Comment by trelau** [comment]
@conda-forge-admin, please restart ci

**Comment by trelau** [comment]
@conda-forge-admin please rerender

**Comment by trelau** [comment]
@conda-forge-admin please rerender

**Comment by trelau** [comment]
@conda-forge-admin, please re-render.

**Comment by looooo** [comment]
The j1 is needed for windows to avoid out of memory abort. Rerendering and setting azure in the conda-forge.yaml won't work for staged recipes.

**Comment by looooo** [comment]
@trelau as far as I know, rerendering is only supported for  feedstocks. Staged recipes should be rebased on the conda-forge/staged-recipes master branch.

**Comment by trelau** [comment]
@looooo thanks for info...i haven't used these tools before (that's probably obvious)...not sure this is best way to test and build for linux...makes me want to have linux on my local windows machine...

**Comment by looooo** [comment]
Hmm, somehow I feel we are turning in circles. I agree that we need to debug the build locally, but first the build should be setup correctly.

**Comment by trelau** [comment]
@looooo i see there is some new differences between what "xcolor" was before and what it is now in the new bindings...seems to be in part causing the linux failures...i might look into a 'dual boot" configuration locally...not sure how else i can reasonably debug

longer term, i wonder if running the bindings on the fly to capture any platform specific types or settings is the way to go...i think this is the approach the CadQuery/pywrap project is taking...seems promising...just more effort required i suppose in the binding generation tool

shall we close this PR? i think things build...just the code needs some work...

**Comment by looooo** [comment]
> @looooo i see there is some new differences between what "xcolor" was before and what it is now in the new bindings...seems to be in part causing the linux failures...i might look into a 'dual boot" configuration locally...not sure how else i can reasonably debug

For debugging the conda-build you can try `conda debug` (equivalent to `conda build` but allows iterating the build.)

> longer term, i wonder if running the bindings on the fly to capture any platform specific types or settings is the way to go...i think this is the approach the CadQuery/pywrap project is taking...seems promising...just more effort required i suppose in the binding generation tool

sounds good! Definitely preferable over iterating the binding code for every new release of occt ; ).

**Comment by looooo** [comment]
`C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\type_traits(1044): fatal error C1060: compiler is out of heap space`
@trelau That's why I switched to -j1 for windows.

**Pull request closed without merging**